source/docs/software/basic-programming/java-units.rst

```diff
@@ -1,5 +1,7 @@
 # The Java Units Library
 
+.. note:: New for 2025: The units library has been refactored to have unit-specific measurement classes instead of a single generic ``Measure`` class.
```

Comment:
I think it would be helpful to have another sentence for the motivation for the change.